### DISCUSSION TITLE
Publii not opening after switch to new computer

### DISCUSSION THREAD

**Discussion by SOLTEK-007** [GetPublii/Publii#1511] [Help] [answered]
I switched to a new computer. I installed Publii on the new computer. I opened Publii and get an error that says, "Publii cannot find your site folder" and when I click, "OK" the app closes. The Publii folder is located on OneDrive, but the problem is that Publii is looking for the OneDrive folder in a path that still contains my old Window's username. Now that I'm on a new PC, my Window's username is different. How do I fix this? Thanks.
![image](https://github.com/GetPublii/Publii/assets/89087831/32b45e6e-df7e-4fe5-980a-b273a2a965d0)

**Comment by dziudek** [comment, marked as answer]
Hi,

Please edit Publii/config/app.config.json file - there is field `"sitesLocation"` which you should update in this case

**Reply by SOLTEK-007** [reply]
I don't see the config folder. 
![image](https://github.com/GetPublii/Publii/assets/89087831/abcf79da-f76d-42c1-a1f0-ca6c3f37901a)

**Reply by dziudek** [reply]
Please look under C:/Users/YOUR_USER/Documents/Publii catalog :)

**Reply by SOLTEK-007** [reply]
@dziudek That worked. Thank you!
